.github/workflows/check-i18n-changes.yml

```diff
@@ -0,0 +1,23 @@
+name: Protect content from manual changes
+
+on: 
+  pull_request:
+    paths:
+    - 'content/**'
+
+jobs:
+  creator-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: 
+        if: github.actor != "nodejs-crowdin" || github.actor != "github-actions"
```

Comment:
Not sure if the `github-actions` actor needs to be allowed, but I don't know what case that is covering